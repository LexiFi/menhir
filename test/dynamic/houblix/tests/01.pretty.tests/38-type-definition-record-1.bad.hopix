type people = { age : int; name : string; firstname : string }