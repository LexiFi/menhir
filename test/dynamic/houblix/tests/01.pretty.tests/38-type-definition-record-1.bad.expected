Line 1, characters 25-26: Error (parsing)
  Syntax error.
